src/pages/Jungmin/Login/InputContainer/InputContainer.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import InputBox from './InputBox';
+import './InputContainer.scss';
+
+class Button extends React.Component {
+  render() {
+    return (
+      <>
+        <button
+          className={
+            !this.props.disabled
+              ? 'loginButton loginButtonOn'
+              : 'loginButton loginButtonOff '
+          }
+          disabled={this.props.disabled}
+          onClick={this.props.goToMain}
+        >
+          로그인
+        </button>
+        <button onClick={this.props.goToSignup} className="forgetPw">
+          회원가입하기
+        </button>
+      </>
+    );
+  }
+}
+
+class Andline extends React.Component {
+  render() {
+    return (
+      <div className="andLine">
+        <div className="line"></div>
+        <div className="and">또는</div>
+        <div className="line"></div>
+      </div>
+    );
+  }
+}
+
+class InputContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: '',
+      password: '',
+      disableId: true,
+      disablePw: true,
+    };
+  }
+
+  componentDidMount() {}
+
+  changeState = e => {
+    const { name, value } = e.target;
+    if (name === 'username') {
+      this.setState({ username: value, disableId: value.indexOf('@') === -1 });
+    } else if (name === 'password') {
+      this.setState({ password: value, disablePw: value.length < 5 });
+    }
+  };
+  goToSignup = () => {
+    this.props.history.push('./signup-jungmin');
+  };
+
+  goToMain = () => {
+    fetch('http://10.58.6.61:8000/users/signin', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.username,
+        password: this.state.password,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => {
+        if (result.access_token) {
+          localStorage.setItem('token', result.access_token);
+          this.props.history.push('/main-jungmin', result);
+        } else {
+          alert('이메일과 비밀번호를 확인해주세요 !');
+        }
+      });
+    this.props.history.push('/main-jungmin', this.state);
+  };
+
+  render() {
+    let logindisable = this.state.disableId || this.state.disablePw;
+    return (
+      <div className="inputContainer">
```

Comment:
```suggestion
  render() {
    const { disableId, disablePw } = this.state
    const logindisable = disableId || disablePw;
    
    return (
      <div className="inputContainer">
```
- 가급적 const 사용해주세요
- 구조분해 할당도 습관적으로 해주세요